code/_onclick/item_attack.dm

```diff
@@ -17,6 +17,10 @@
 	user.changeNext_move(CLICK_CD_MELEE)
 	if(stat == DEAD && !isnull(butcher_results)) //can we butcher it?
 		if(istype(I, /obj/item/weapon/kitchen/knife))
+			if(istype(I, /obj/item/weapon/kitchen/knife/hunting))
```

Comment:
I just tried, and couldn't get it to work, after multiple tries, so I'll just leave this as-is.